cmd/frontend/graphqlbackend/git_commit.go

```diff
@@ -68,7 +68,16 @@ type GitCommitResolver struct {
 // commit will be loaded lazily as needed by the resolver. Pass in a commit when
 // you have batch-loaded a bunch of them and already have them at hand.
 func NewGitCommitResolver(db database.DB, gsClient gitserver.Client, repo *RepositoryResolver, id api.CommitID, commit *gitdomain.Commit) *GitCommitResolver {
+	var repoName api.RepoName
+	if repo != nil {
+		repoName = repo.RepoName()
+	}
+
 	return &GitCommitResolver{
+		logger: log.Scoped("gitCommitResolver", "resolve a specific commit").
+			With(log.Object("commit",
+				log.String("repo", string(repoName)),
+				log.String("commitId", string(id)))),
```

Comment:
```suggestion
		logger: log.Scoped("gitCommitResolver", "resolve a specific commit").
			With(log.Object("commit",
				log.String("repo", string(repoName)),
				log.String("commitID", string(id)))),
```

In Go, it's suggested to use `ID` instead of `Id` when capitalization is used.

Reply:
updated